src/main/java/org/assertj/core/util/diff/Delta.java

```diff
@@ -136,6 +136,7 @@ int lineNumber() {
   }
 
   String formatLines(List<T> lines) {
+    //TODO see what we should use here
     return STANDARD_REPRESENTATION.format(lines, DEFAULT_START, DEFAULT_END, ELEMENT_SEPARATOR_WITH_NEWLINE, "   ");
```

Comment:
This was the only place that I could not remove the `STANDARD_REPRESENTATION` usage. Maybe we need to move this formatting to some other class instead.